proxy/src/serve_tasks.rs

```diff
@@ -35,8 +37,10 @@ pub(crate) fn router(client: &SamplyHttpClient) -> Router {
     };
     Router::new()
         // We need both path variants so the server won't send us into a redirect loop (/tasks, /tasks/, ...)
-        .route("/v1/tasks", any(handler_tasks))
-        .route("/v1/tasks/*path", any(handler_tasks))
+        .route("/v1/tasks", get(handler_tasks).post(handler_tasks))
+        .route("/v1/tasks/:task_id/results", get(handler_tasks))
+        .route("/v1/tasks/:task_id/results/stream", get(handler_tasks_stream))
```

Comment:
Same comment as with the broker.

Reply:
Fixed in https://github.com/samply/beam/commit/403c5e0a876aa0f7a337a559af09f7803b633025